### PR TITLE
Feature/launch browser on run

### LINKED ISSUE #659: Open up hypervisorUI when starting hypervisor from CLI
**Feature description**

Synth requested that we add a feature that opens up the local hypervisor when starting a hypervisor from CLI. This feature already is implemented by [skycoin/skycoin](https://github.com/skycoin/skycoin/blob/c4e1328ede14c8241449a66f587d3ab489715f4b/src/skycoin/skycoin.go#L271)  for reference. This, together with the improvement here #658 should make management of a local node pretty smooth.

### PR DESCRIPTION
**Pull request by i-hate-nicknames**
Did you run `make format && make check`?

Fixes #659

 Changes:	
- Add --launch-browser flag that starts default user browser on run (hypervisor only)

How to test this PR:
Run skywire-visor with `--launch-browser` flag and hypervisor settings in config.

### CODE REVIEW

**Review by jdknives — APPROVED** [review]
Good job. Works smoothly.

File: cmd/skywire-visor/commands/root.go [hunk 1]
```diff
@@ -48,6 +50,7 @@ func init() {
 	rootCmd.Flags().StringVar(&pprofAddr, "pprofaddr", "localhost:6060", "pprof http port if mode is 'http'")
 	rootCmd.Flags().StringVarP(&confPath, "config", "c", "", "config file location. If the value is 'STDIN', config file will be read from stdin.")
 	rootCmd.Flags().StringVar(&delay, "delay", "0ns", "start delay (deprecated)") // deprecated
+	rootCmd.Flags().BoolVar(&launchBrowser, "launch-browser", false, "start delay (deprecated)")
```
Comment (jdknives) [inline cmd/skywire-visor/commands/root.go]: Please update the description here. Looks like it was copied.

File: cmd/skywire-visor/commands/root.go [hunk 2]
```diff
@@ -249,3 +256,27 @@ func initConfig(mLog *logging.MasterLogger, args []string, confPath string) *vis
...
+	}
+	if addr[:4] != "http" {
+		if conf.Hypervisor.EnableTLS {
+			addr = "https://" + addr
+		} else {
+			addr = "http://" + addr
+		}
+	}
+	go func() {
+		time.Sleep(1 * time.Second)
```
Comment (Darkren) [inline cmd/skywire-visor/commands/root.go]: This timeout might be sufficient in most cases, but we can make it more reliable if we send a request to `/api/health`, it should respond when hypervisor is ready. What do you think of it?

**Review by Darkren — COMMENTED** [review]
Great job! I have just one suggestion on this

File: cmd/skywire-visor/commands/root.go (hunk 2 above)
Comment (i-hate-nicknames) [reply cmd/skywire-visor/commands/root.go]: Thanks, I had an uneasy feeling when I was adding this timeout

Comment (jdknives) [reply cmd/skywire-visor/commands/root.go]: Just testing the webhook.

Comment (jdknives) [reply cmd/skywire-visor/commands/root.go]: Testing again.

Comment (jdknives) [reply cmd/skywire-visor/commands/root.go]: And again.

Comment (jdknives) [reply cmd/skywire-visor/commands/root.go]: And again

Comment (jdknives) [reply cmd/skywire-visor/commands/root.go]: And again

**Pull request merged**